src/Components/MessageState/NoRecsState.js

```diff
@@ -0,0 +1,59 @@
+import {
+  EmptyState,
+  EmptyStateVariant,
+} from '@patternfly/react-core/dist/js/components/EmptyState/EmptyState';
+
+import CubesIcon from '@patternfly/react-icons/dist/js/icons/cubes-icon';
+import { EmptyStateBody } from '@patternfly/react-core/dist/js/components/EmptyState/EmptyStateBody';
+import { EmptyStateIcon } from '@patternfly/react-core/dist/js/components/EmptyState/EmptyStateIcon';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Title } from '@patternfly/react-core/dist/js/components/Title/Title';
+
+const NoRecsState = ({
+  className,
+  children,
+  icon,
+  iconClass,
+  iconStyle,
+  text,
+  title,
+  variant,
+  link,
+  linktext,
+}) => (
+  <EmptyState className={className} variant={variant}>
```

Comment:
Any reason to keep it separately? Looks like can be added to https://github.com/RedHatInsights/ocp-advisor-frontend/blob/master/src/Components/MessageState/EmptyStates.js